security/file_integrity_operator/file-integrity-operator-troubleshooting.adoc

```diff
@@ -54,14 +54,14 @@ Run:
 +
 [source,terminal]
 ----
-$ oc get pods -lapp=aide-ds-$(<FIO_NAME>)
+$ oc -n openshift-file-integrity get pods -lapp=worker-fileintegrity
```

Comment:
sorry, my mistake.
Should be:
oc -n openshift-file-integrity get pods -lapp=aide-worker-fileintegrity